spyder/plugins/layout/container.py

```diff
@@ -106,6 +107,8 @@ def setup(self):
             self._fullscreen_action.setEnabled(False)
             self._fullscreen_action.setToolTip(_("For fullscreen mode use the "
                                                  "macOS built-in feature"))
+        if is_wsl():
+            self._fullscreen_action.setEnabled(False)
```

Comment:
```suggestion
            self._fullscreen_action.setEnabled(False)
            self._fullscreen_action.setToolTip(
                _("Fullscreen mode doesn't work reliably on WSL")
            )
```

Let's add a tooltip to explain why the action is disabled, as we do for Mac.